### PR TITLE
Adding NOTICE file

### PR DESCRIPTION
**Pull request by maneta**
any ASL 2.0 licensed stuff must have a NOTICE file

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+   Red Hat 3scale API Management Operations
+   Copyright (c) 2010-2016 3scale, Inc
+   Copyright (c) 2016-2018 Red Hat, Inc.
```
Comment (unleashed) [inline NOTICE]: Maybe just use the latter line and 2019 as the year?

**Review by unleashed — APPROVED** [review]
only a minor nit, read the comments

File: NOTICE (hunk 1 above)
Comment (maneta) [reply NOTICE]: sure!

Comment (maneta) [reply NOTICE]: solved on 449bbdc04398a672e836e3e2970e735dc936d2b4

**Pull request merged**